sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/Docs/WebPubSubServiceClient.xml

```diff
@@ -55,12 +55,12 @@ This sample shows how to call SendToAllAsync with all parameters and request con
 WebPubSubServiceClient client = new WebPubSubServiceClient("<Endpoint>", "<Hub>");
 
 using RequestContent content = RequestContent.Create(File.OpenRead("<filePath>"));
-Response response = await client.SendToAllAsync(content, new ContentType("application/json"), excluded: new string[] { "<excluded>" }, filter: "<filter>");
+Response response = await client.SendToAllAsync(content, new ContentType("application/json"), excluded: new string[] { "<excluded>" }, filter: "<filter>", messageTtlSeconds: 1234);
```

Comment:
the max allowed is 300, sounds that we shouldn't use 1234 as a sample number https://github.com/Azure/azure-rest-api-specs/blob/356aa5174e8eec6ed904bf5ff104595aec8c0411/specification/webpubsub/data-plane/WebPubSub/stable/2024-01-01/webpubsub.json#L401